Maestro/subscriptions.json

```diff
@@ -519,6 +519,31 @@
         "https://github.com/dotnet/core-setup/blob/release/1.1.0/**/*"
       ],
       "action": "core-setup-pipebuild-release-1.1.0"
+    },
+    // Update dependencies in WCF release/2.0.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/standard/release/2.0.0/Latest.txt"
```

Comment:
It would make sense to me to include a Core-Setup trigger in this PR, too, if that's coming soon to WCF. (One less dotnet/versions PR to make.) These paths only affect when the auto-upgrade build is triggered--dependencies.props takes it from there and upgrades everything it knows about.